VRDR/DeathRecord.cs

```diff
@@ -5819,6 +5819,46 @@ public string DeathTime
             }
         }
 
+        /// <summary>DateOfDeathDeterminationMethod.</summary>
+        /// <value>method. A Dictionary representing a code, containing the following key/value pairs:
+        /// <para>"code" - the code</para>
+        /// <para>"system" - the code system this code belongs to</para>
+        /// <para>"display" - a human readable meaning of the code</para>
+        /// </value>
+        /// <example>
+        /// <para>// Setter:</para>
+        /// <para>Dictionary&lt;string, string&gt; code = new Dictionary&lt;string, string&gt;();</para>
+        /// <para>code.Add("code", "exact");</para>
+        /// <para>code.Add("system", CodeSystems.DateOfDeathDeterminationMethods);</para>
+        /// <para>code.Add("display", "exact");</para>
+        /// <para>ExampleDeathRecord.DateOfDeathDeterminationMethod = code;</para>
+        /// <para>// Getter:</para>
+        /// <para>Console.WriteLine($"Date of Death Determination Method: {ExampleDeathRecord.DateOfDeathDeterminationMethod['display']}");</para>
+        /// </example>
+
+       [Property("DateOfDeathDeterminationMethod", Property.Types.Dictionary, "Death Investigation", "Date of Death Determination Method.", true, IGURL.DeathDate, true, 25)]
+       [FHIRPath("Bundle.entry.resource.where($this is Observation).where(code.coding.code='81956-5').method", "")]
+        public Dictionary<string, string> DateOfDeathDeterminationMethod
+        {
+            get
+            {
+                if (DeathDateObs != null && (DeathDateObs.Method as CodeableConcept) != null)
+                {
+                    return CodeableConceptToDict(DeathDateObs.Method);
+                }
+                return EmptyAddrDict();
```

Comment:
Why are you returning an EmptyAddrDict()? Don't you want EmptyCodeableDict()?

Reply:
Good catch....